components/automate-ui/src/app/services/projects-filter/projects-filter.reducer.ts

```diff
@@ -36,22 +45,35 @@ export function projectsFilterReducer(
   switch (action.type) {
 
     case ProjectsFilterActionTypes.LOAD_OPTIONS: {
-      return set('optionsLoadingStatus', LoadingStatus.loading, state);
+      return set('optionsLoadingStatus', EntityStatus.loading, state);
     }
 
     case ProjectsFilterActionTypes.LOAD_OPTIONS_SUCCESS: {
+      const sortedOptions = action.payload
+        .filter(o => o.value !== UNASSIGNED_PROJECT_ID)
+        .sort((a, b) => {
+          const opts = { numeric: true, sensitivity: 'base' };
+          return a.label.localeCompare(b.label, undefined, opts)
+            || a.label.localeCompare(b.label, undefined, { numeric: true });
+        }
+        );
+      const unassignedProject = find(['value', UNASSIGNED_PROJECT_ID], action.payload);
+      if (unassignedProject) {
+        sortedOptions.push(unassignedProject);
+      }
```

Comment:
The above forces unassigned to be at the bottom of the list.

Reply:
We could keep this switch case simple by sorting the success payload instead: 
https://github.com/chef/automate/commit/4540debd3df47ef7a3121f91a4f6b45bda3873f0 <- working example but breaks the reducer tests

Moving these transformations into the effect also kinda helps clarify what's being described in [this previous comment](https://github.com/chef/automate/pull/172#discussion_r278750284). Or perhaps we could do the inverse by changing the success payload in a way that would allow all this [merging and sorting](https://github.com/chef/automate/commit/4540debd3df47ef7a3121f91a4f6b45bda3873f0#diff-4242aab319947ab25a57f84b34fb4240R47) to happen in the reducer file and keep the effect file super simple instead 🤔 

I'm +1 regardless though since this all works as advertised and is well-tested 👍👍

Reply:
Appreciate the suggestions--I will take a look at what you suggested.